### PR TITLE
Create jmuku.md

### PR DESCRIPTION
**Pull request by jmuku**
## Describe your project in 1-3 sentences

Holographic Virtual assistant and smart mirror
## Hack Club Slack handle
Mukesh Kumar

## Checklist before submitting proposal:

- [ yes] I am a current high school student (or younger).
- [ yes] I have submitted an RSVP at <https://hackclub.com/winter>
- [ yes] I am a member of the Hack Club Slack (Join at <https://hackclub.com/slack>)
- [ yes] I have self-reviewed my project proposal.

### CODE REVIEW

**Comment by sohamdagoat** [comment]
## Hack Club Slack handle
Soham Patel


## Checklist before submitting proposal:

- [ yes] I am a current high school student (or younger).
- [ yes] I have submitted an RSVP at <https://hackclub.com/winter>
- [ yes] I am a member of the Hack Club Slack (Join at <https://hackclub.com/slack>)
- [ yes] I have self-reviewed my project proposal.

**Comment by jmuku** [comment]
@exu3 can you please chek  our project now

File: jmuku.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
+---
+name: "@jmuku"
```
Comment (bellesea) [inline jmuku.md:2]:
```suggestion
name: ["@jmuku", "@sohamdagoat"]
```

**Comment by jmuku** [comment]
both projects need a speaker and microphone but i believe the kit only has
one? do y'all have a plan for this?
Soham's already got a jbl speaker with built in mic, so he doesn't need the
pi speakers

do you have a screen for the holographic assistant?
yes, its included in the kit i linked (we will only be making ine of these)

do you have any plans for modifying the holographic assistant in anyway?
yeah, the housing for it is 3d printed, we plan on modifying the 3d print
files in fusion 360. We have a 3d printer available.

what are your plans for the enclosure for the holographic assistant?
written above!

when you mention already having the power supply/sd cards etc. do you mean
as part of the kit? if not, would it be better to just purchase the pi
outside of the kit?
I meant it as part of the kit, sorry for the confusion.

are you following any guides for the mirror?
what are your plans for the framing of the smart mirror?
yeah we'll be following youtube videos for making it. We have a frame which
is similarly sized to our monior. We will be using that

**Comment by jmuku** [comment]
Added all the questions that you asked to the file too

**Review by bellesea — APPROVED** [review]
@jmuku and @sohamdagoat this is really really awesome! an awesome member of @hackclub/bank-ops will be reaching out with next steps :)

approving the project to move it along but could you please make the following edits too:
- instead of questions/answers in the file could you please just elaborate in the relevant portion of the file?
- for the screen for the assistant, it looks like the screen from the kit would be super small.. if that works for you it's 100% ok but if you decide later on you need a bigger screen i hope you can just update your file so we can have an updated "build file"

thank you so much!

**Comment by jmuku** [comment]
@bellesea did everything you asked🙃

**Review by MelSmith104 — COMMENTED** [review]
hi @jmuku and @sohamdagoat !! this project looks awesome! it looks like it's going to be so much fun to build together! im dm'ing you both on slack right now so that we can chat about next steps!!

**Comment by zhingzhangzhung** [comment]
Hi im justin and me, mukesh and soham go to the same school. Since soham couldn't participate this time, I'll be taking his spot

My Slack id is: Justin Zhang
[ yes] I am a current high school student (or younger).
[ yes] I have submitted an RSVP at https://hackclub.com/winter
[ yes] I am a member of the Hack Club Slack (Join at https://hackclub.com/slack)
[ yes] I have self-reviewed my project proposal.

**Comment by jmuku** [comment]
@MelSmith104 i changed everything and added justin in replacement of soham

**Review by MelSmith104 — APPROVED** [review]

**Pull request merged**